go/vt/sqlparser/sql.y

```diff
@@ -466,6 +466,7 @@ func yySpecialCommentMode(yylex interface{}) bool {
 %type <columnType> column_type  column_type_options json_table_column_options on_empty
 %type <columnType> int_type decimal_type numeric_type time_type char_type spatial_type
 %type <sqlVal> length_opt column_comment ignore_number_opt
+%type <sqlVal> var_length_opt column_comment ignore_number_opt
```

Comment:
There's some duplication here that would be good to simplify: 

```suggestion
%type <sqlVal> var_length_opt length_opt column_comment ignore_number_opt
```

Reply:
I don't know if this possible in yacc since I'm new with it. Thanks for the feedback! 🍻